.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "cosmos-sdk"]
 	path = third_party/cosmos-sdk
 	url = https://github.com/cosmos/cosmos-sdk.git
+
+[submodule "wasmd"]
```

Comment:
Why do we need the submodule?

Reply:
Simulation tests depend on some `wasm` files from repository.

Reply:
https://github.com/crypto-org-chain/chain-main/pull/780/files#diff-6377d84e477b544abf9150c43e9f581ac6aaa209cee8f26961187cebe0d200a0

Reply:
the `reflect.wasm` contract, if that's the only thing we need, maybe better to just copy it?

Reply:
Then we'll have to manually manage the file with every release of `wasmd`.